### PR TITLE
Various definition touchups

### PR DESCRIPTION
**Pull request by john-kurkowski**
Clarifications to the following:
- Observer
- Computed property
- Module
- Initializer
- Integration test

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -189,18 +189,21 @@ <h2 id="project_tagline">A simple glossary for Ember terms and concepts</h2>
...
 
-<p>Triggers a function when a primitive property changes.</p>
+<p>Immediately triggers a side-effecting function, when a primitive property changes.</p>
 
 <div class="highlight highlight-js"><pre>Ember.<span class="pl-c1">Object</span>.extend({
-  valueObserver<span class="pl-k">:</span> Ember.observer(<span class="pl-s"><span class="pl-pds">'</span>value<span class="pl-pds">'</span></span>, <span class="pl-k">function</span>() {
-    <span class="pl-c">// Executes whenever the "value" property changes</span>
+  onIsFormValidChanged<span class="pl-k">:</span> Ember.observer(<span class="pl-s"><span class="pl-pds">'</span>isFormValid<span class="pl-pds">'</span></span>, <span class="pl-k">function</span>() {
+    <span class="pl-c">// Executes whenever the "isFormValid" property changes</span>
+    if (!this.get('isFormValid')) {
```
Comment (john-kurkowski) [inline index.html]: I don't know how to syntax highlight this code snippet, like you've done elsewhere in this file. This index.html almost looks compiled. But I couldn't find the source in any branch. Can you help highlight this?

**Comment by kylecoberly** [comment]
Hey John! Sorry for the slow response, I didn't get the notification for this. It is actually compiled via GH pages. I'll dig into this as soon as possible.

**Comment by kylecoberly** [comment]
So instead of authoring directly into the codebase, go to Settings -> Github Pages -> Launch Automatic Page Generator and update the markdown file. It should generate the commit for you which, can be PRed.

**Comment by john-kurkowski** [comment]
Done!

Weird they only expose version control for the compiler output and not the source. Seems like compiler output shouldn't even be in Git. They don't do that for my personal GitHub Pages-hosted Jekyll site, for example. But I digress.

Also their generator replaced your name with mine. I reverted that by hand.

**Pull request merged**